webapp-django/crashstats/crashstats/static/crashstats/js/socorro/utils.js

```diff
@@ -230,7 +230,23 @@
                 }
 
                 return params;
-            }
+            },
+            sortResults: function (results, container, query) {
+                if (query.term) {
+                    return results.sort(function (a, b) {
+                        if (a.text.length > b.text.length) {
```

Comment:
I still don't get it. Well, I understand what you've done and what the code does but I don't understand the reasoning. 
Here's what it looks like:
<img width="1271" alt="screen shot 2016-08-19 at 5 19 24 pm" src="https://cloud.githubusercontent.com/assets/26739/17824644/3d24cd24-6631-11e6-837f-08a70b5535bd.png">

And here's what it would look like with no explicit sorting (good old default alpha sort)
<img width="673" alt="screen shot 2016-08-19 at 5 21 03 pm" src="https://cloud.githubusercontent.com/assets/26739/17824662/626138a2-6631-11e6-81f5-975c9b165466.png">

I think I have a better solution. Instead of sorting by length, we sort by how left-most the match is. And in case of ties, sort alphabetically. 
<img width="680" alt="screen shot 2016-08-19 at 5 27 53 pm" src="https://cloud.githubusercontent.com/assets/26739/17824859/69c22ab0-6632-11e6-8d09-f4cd48a25e4e.png">
...and...
<img width="731" alt="screen shot 2016-08-19 at 5 29 23 pm" src="https://cloud.githubusercontent.com/assets/26739/17824876/800a301a-6632-11e6-9d6c-74f88b2645c8.png">

Do you think that looks better? I think I do.